components/RecipeDetails.js

```diff
@@ -2,25 +2,33 @@ import styled from "styled-components";
 import Image from "next/image";
 import Link from "next/link";
 
-export default function RecipeDetails({ recipeData }) {
+export default function RecipeDetails({ recipeDetails }) {
   return (
     <RecipeCard>
-      <h1> {recipeData.name}</h1>
+      <h1> {recipeDetails.name}</h1>
       <StyledImage
-        src={recipeData.imageURL}
-        alt={recipeData.name}
+        src={
+          recipeDetails.imageURL
+            ? recipeDetails.imageURL
+            : "/recipe-images/generic-food-3.png"
+        }
```

Comment:
```suggestion
        src={
          recipeDetails.imageURL ?? "/recipe-images/generic-food-3.png"
        }
```

should do the same :)

Reply:
Thank you thats brilliant!

Reply:
Unfortunately we were not able to use this because it broke everything. A empty string would be there and the image component would have a missing src=.